doc/.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!-- Thanks for reporting an issue to axe-core. Please provide all necessary info to reproduce the issue. Without adequate details, your issue may be closed without investigation. 
+
+If you’re reporting a bug, include a description of the issue and a page or code snippet where it can be reproduced. Please make sure you have tested this with the latest version of axe-core. When proposing a new rule, please use our [rule template](https://github.com/dequelabs/axe-core/blob/develop/doc/rule-proposal.md).
+
+Describe what the desired behavior would be, and how it applies to "Accessibility Supported" https://github.com/dequelabs/axe-core/blob/develop/doc/accessibility-supported.md
```

Comment:
Not sure what you're asking for with "accessibility supported"?

Reply:
I'm asking that people read it and apply it to their change. We have to mention it all the time.